.github/workflows/checks.yml

```diff
@@ -13,26 +13,15 @@ jobs:
 
     steps:
       - name: Checkout code
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
 
       - name: Set up Node.js
-        uses: actions/setup-node@v3
+        uses: actions/setup-node@v4
         with:
-          node-version: "20"
+          node-version: "22"
 
-      - name: Install dependencies
-        run: npm ci
+      - name: get prettier prettier-plugin-svelte
+        run: npm i --save-dev prettier-plugin-svelte prettier
```

Comment:
Hvorfor ikke npm ci? Med den kommandoen er vi sikker på at vi får nøyaktig den samme versjonen av programmene som vi bruker lokalt, mens her er det en mulighet for å mangle noe, eller for å ha feil versjon.

Reply:
Godt poeng, var litt på automatikk